pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/TransformFunction.java

```diff
@@ -319,6 +319,13 @@ private static TransformStep newComputeFieldFunction(ComputeConfig config) {
                 throw new IllegalArgumentException(
                     "Duplicate compute field name detected: " + field.getName());
               }
+              if (field.getType() == ComputeFieldType.DATE
```

Comment:
Is there a need to update the type inference logic to explicitly not infer `DATE` for primitive types?

Reply:
We don't have any function that would infer a DATE atm. And I believe there won't ever be.